### PR TITLE
Upgrade to Latest dependencies in the blueprints

### PR DESCRIPTION
**Pull request by NullVoxPopuli**

### CODE REVIEW

**Comment by NullVoxPopuli** [comment]
merging for emergency release? it's been a while. :-\
there is no octane blueprint in ember-cli, so the released version of this is somewhat out of date.

File: packages/@ember/octane-addon-blueprint/files/package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     "@ember/edition-utils": "^1.1.1",
     "@ember/optional-features": "^1.0.0",
     "@glimmer/component": "^0.14.0-alpha.12",
```
Comment (buschtoens) [inline packages/@ember/octane-addon-blueprint/files/package.json]: btw `@glimmer/component` is now at [`1.0.0-beta.3`](https://www.npmjs.com/package/@glimmer/component?activeTab=versions). Was this intentionally left out?

Comment (NullVoxPopuli) [reply packages/@ember/octane-addon-blueprint/files/package.json]: Whoops. I just did yarn upgrade latest interactive

Comment (NullVoxPopuli) [reply packages/@ember/octane-addon-blueprint/files/package.json]: I forgot that glimmer componet was still in beta, and yarn upgrade only does real releases

**Pull request merged**